mlflow/server/auth/client.py

```diff
@@ -20,7 +20,9 @@
 
 class AuthServiceClient:
     """
-    Client of an MLflow Tracking Server that enabled the default auth plugin.
+    Client of an MLflow Tracking Server that enabled the default basic authentication plugin.
+    It is recommended to use ``mlflow.server.get_app_client()`` to instantiate this class.
```

Comment:
Is it possible to make `mlflow.server.get_app_client()` a link?

Reply:
sure, i added the docs for it